src/routes/+page.svelte

```diff
@@ -53,7 +53,7 @@
               blurhash={item.heroImage.imageSource?.blurhash ?? undefined}
               width={item.heroImage.imageSource.width ?? undefined}
               height={item.heroImage.imageSource.height ?? undefined}
-              fit
+              sizeType="card"
```

Comment:
Thanks for adding this! Can you remove the `TODO` comment before you merge?
https://github.com/la-ldaf/ldaf-site/blob/da1ce05ce0df3677a08272abb7bb628e76fb5087/src/routes/%2Bpage.svelte#L47